lib/create-matcher/match-object.js

```diff
@@ -6,6 +6,11 @@ var typeOf = require("@sinonjs/commons").typeOf;
 var deepEqualFactory = require("../deep-equal").use;
 
 var isMatcher = require("./is-matcher");
+
+var keys = Object.keys;
+var getOwnPropertySymbols = Object.getOwnPropertySymbols;
+var concat = Array.prototype.concat;
```

Comment:
Can you use the cached `concat` from `@sinonjs/commons` here? See line 3, where we do this for `every`.